sdk/src/main/java/com/qonversion/android/sdk/internal/QonversionInternal.kt

```diff
@@ -7,21 +7,32 @@ import com.qonversion.android.sdk.dto.CacheLifetime
 import com.qonversion.android.sdk.dto.Environment
 import com.qonversion.android.sdk.dto.LogLevel
 import com.qonversion.android.sdk.dto.UserProperty
+import com.qonversion.android.sdk.dto.User
 import com.qonversion.android.sdk.internal.cache.CacheLifetimeConfig
 import com.qonversion.android.sdk.internal.cache.InternalCacheLifetime
+import com.qonversion.android.sdk.internal.exception.QonversionException
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
 import com.qonversion.android.sdk.internal.di.DependenciesAssembly
+import com.qonversion.android.sdk.internal.user.controller.UserController
 import com.qonversion.android.sdk.internal.userProperties.controller.UserPropertiesController
 
 internal class QonversionInternal(
     config: QonversionConfig,
     private val internalConfig: InternalConfig,
-    dependenciesAssembly: DependenciesAssembly
+    dependenciesAssembly: DependenciesAssembly,
+    private val scope: CoroutineScope = CoroutineScope(Dispatchers.IO)
 ) : Qonversion {
 
-    @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+    @VisibleForTesting
     val userPropertiesController: UserPropertiesController =
         dependenciesAssembly.userPropertiesController()
 
+    @VisibleForTesting
+    val userController: UserController =
+        dependenciesAssembly.userController()
```

Comment:
I offer to write in one line if it fits
```suggestion
    val userController: UserController = dependenciesAssembly.userController()
```